templates/singleton/boil_types.go.tpl

```diff
@@ -75,8 +160,9 @@ It only titlecases the EnumValue portion if it's snake-cased.
 // Enum values for {{if $isNamed}}{{$name}}{{else}}{{$table.Name}}.{{$col.Name}}{{end}}
 const (
 	{{- range $val := $vals -}}
+		{{- $valStripped := stripWhitespace $val -}}
```

Comment:
Why's this necessary? It hasn't been in the past. Should this make it in as a separate fix?

Reply:
because if the enum value has a space in it (which is allowed in mysql at least, perhaps other dbs) then the generated code is invalid.